src/eddington_gui/app.py

```diff
@@ -345,6 +346,24 @@ def __calculate_fit_result(self):
                 title="Fit result error", message=str(error),
             )
             raise error
+   
+    def choose_sheet(self):
+        """Automatically choose the first valid sheet"""
+        for sheet in self.input_file_box.sheets_options:
+            try:
+                self.data_columns_box.fit_data = FitData.read_from_excel(
+                    Path(self.input_file_box.file_path),
+                    sheet)
+                self.input_file_box.children[1].children[1].value = sheet
```

Comment:
The EddingtonGUI class should know which children InputFileBox has. Use [python property](https://www.programiz.com/python-programming/property) in order to set the sheet using the following syntax:
```python
self.input_file_box.selected_sheet = sheet
```